shell/layouts/default.vue

```diff
@@ -44,7 +45,7 @@ export default {
     FixedBanner
   },
 
-  mixins: [PageHeaderActions, Brand],
+  mixins: [PageHeaderActions, Brand, BrowserTabVisibility],
```

Comment:
This has been added to the default and home layouts, it'll need also adding to plain. I don't think we need to cover plain, and definitely no unauthenticated or error.

Reply:
added to `plain`